test/test_kernel_launch.jl

```diff
@@ -0,0 +1,138 @@
+include("common.jl")
+
+import Chmy.Architectures: deepmap!, disable_task_sync!, enable_task_sync!
```

Comment:
I don't get why besides `deepmap!`, I need to also import the other functions here as they should be exported

Reply:
You are adding methods to `disable_task_sync!`, `enable_task_sync!`. To do so you need to import them, or do e.g. 
```julia
Chmy.Architectures.disable_task_sync!(::Any) = nothing
```

Reply:
Ok, this one makes sense to me, yes.